doc/nrf/includes/boardname_tables/sample_boardnames.txt

```diff
@@ -305,3 +305,21 @@ Set used by sample (nRF RPC Entropy)
 
 
 .. set21_end
+
+Set used by samples (low power UART driver)
+
+.. set22_start
+
++--------------------------------+-----------+------------------------------------------------+-------------------------------+
+|Hardware platforms              |PCA        |Board name                                      |Build target                   |
++================================+===========+================================================+===============================+
+|:ref:`nRF9160 DK <ug_nrf9160>`  |PCA10090   |:ref:`nrf9160dk_nrf9160 <nrf9160dk_nrf9160>`    |``nrf9160dk_nrf9160ns``        |
```

Comment:
The build target here does not match the `platform_whitelist` in `samples/peripheral/lpuart/sample.yaml`. I'm not sure if it should, though.